### DISCUSSION TITLE
Consuming data from remote GraphQL endpoint

### DISCUSSION THREAD

**Discussion by 3goats** [blitz-js/blitz#2302] [General]
Hi, is there was to include a query to a remote GraphQL endpoint in Blitz? I need to include remote data in my app potentially with the data provided by the local Blitz database.

**Comment by flybayer** [comment]
Are you wanting to query that endpoint from the browser or from the server?

**Comment by 3goats** [comment]
Good question. Not sure at this stage where best to do this. I assume it’s possible to do both so I would like to try it both ways if possible. I guess doing it from the browser comes with its own CORS related issues though?

**Reply by flybayer** [reply]
Correct. On the server, you can query any API from inside a Blitz query or mutation resolver. There's nothing special about that, normal node.js stuff.

From the browser, you can do user your favorite graphql client like Apollo, urql, or react-query (which is already installed because it comes with blitz). Again there's nothing special about blitz to this. You'd do it like you would in any react app

**Comment by 3goats** [comment]
Thanks - I’ll take look. Are there any examples for this anywhere ?

**Reply by flybayer** [reply]
Not that I'm aware of for Blitz, but there's tons of examples for doing this with create-react-app or next.js, both of which you can reference.
